packages/core/datadog-configuration.gradle

```diff
@@ -0,0 +1,41 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+def getConfigurationFilePath() {
+    if (project.hasProperty("datadog")) {
+        if (project.datadog.configurationFilePath) {
+            return project.datadog.configurationFilePath
+        }
+    }
+    return '../../datadog-configuration.json'
+}
+
+def getBuildAssetsDirectory() {
+    return project.layout.buildDirectory
+        .dir("generated/assets/datadog/configuration")
+}
+
+/*
+ * Adds the copied configuration file to the assets of the app for every sourceSet.
+ */
+task addDatadogConfigurationToAssets() {
+    android.sourceSets.forEach { source ->
+        source.assets.srcDir(getBuildAssetsDirectory())
```

Comment:
This triggers this popup in Android studio when syncing Gradle:
![image](https://github.com/DataDog/dd-sdk-reactnative/assets/8973379/a58f5f24-a7f8-40e3-a4a7-eab6b5e8e570)

I'm looking into how to remove it safely

Reply:
> As there can be multiple production source sets anyway, I prefered to copy it for every source anyway in a separate directory.

Yes, this is the right approach I think.